docs/conf.py

```diff
@@ -70,6 +71,9 @@
     "plasmapy_sphinx",
 ]
 
+# Intersphinx generates automatic links to the documentation of objects
+# in other packages. When mappings are removed or added, please update
+# the documentation guide in the development guide.
```

Comment:
Reference the exact file and section that should be updated, so a user does not have to figure that out on their own.